src/sagemaker/fw_utils.py

```diff
@@ -148,7 +148,15 @@
 ]
 
 
-TORCH_DISTRIBUTED_SUPPORTED_FRAMEWORK_VERSIONS = ["1.11", "1.11.0"]
+TORCH_DISTRIBUTED_TRAINIUM_SUPPORTED_FRAMEWORK_VERSIONS = [
+    "1.11",
+    "1.11.0",
+    "1.12",
```

Comment:
Are we sure Trainium supports > PT 1.11?

Reply:
in their documentation, the notes said "torchrun support starting from 1.11", but I will verify this with Trainium team.

Reply:
verified with Trainium team, Trainium support all of these versions.